terminal/display.go

```diff
@@ -1,11 +1,9 @@
-// +build !windows
-
 package terminal
 
-import (
-	"fmt"
-)
+type EraseLineMode int
 
-func EraseInLine(mode int) {
-	fmt.Printf("\x1b[%dK", mode)
-}
+const (
+	ERASE_LINE_END   EraseLineMode = iota
+	ERASE_LINE_START EraseLineMode = iota
```

Comment:
Sorry to be a stickler, but do you mind removing the repeated `EraseLineMode = iota?